chainer/functions/connection/deconvolution_2d.py

```diff
@@ -325,21 +325,49 @@ def deconvolution_2d(x, W, b=None, stride=1, pad=0,
                      outsize=None, use_cudnn=True, deterministic=False):
     """Two dimensional deconvolution function.
 
-    This is an implementation of two-dimensional deconvolution.
+    This is an implementation of two-dimensional deconvolution (alse known as
+    transposed convolution).
```

Comment:
Yes, they said that name of deconvolution is not good.
TensorFlow have named this functions as `conv2d_transpose`.
So, it is better to rename deconvolution functions, but it will break
the backward compatiblity.
Please ask if we should rename the functions or not.